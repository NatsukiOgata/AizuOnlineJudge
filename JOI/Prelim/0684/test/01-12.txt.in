99 99
84 65 39 36 28 34 98 64 41 69 55 83 92 56 66 99 87 15 16 87 6 32 52 73 55 40 19 39 100 37 40 10 89 54 39 26 89 50 35 82 20 89 7 7 50 9 81 64 52 15 48 55 56 82 38 82 46 37 84 38 96 56 39 79 93 7 51 71 2 17 39 66 54 80 19 47 71 100 8 48 33 12 5 68 62 36 11 59 59 13 5 12 15 19 83 73 9 27 25
76 18 44 92 29 41 60 81 66 42 25 13 40 99 21 71 40 60 96 68 34 2 86 76 4 90 70 26 67 48 1 12 76 33 13 4 67 39 13 29 16 90 3 62 96 85 17 44 78 73 57 46 45 23 20 73 65 78 100 30 11 82 83 74 77 5 62 13 24 30 52 45 82 82 77 29 25 75 41 80 92 51 24 43 68 43 12 97 32 60 56 16 77 82 95 50 6 62 9
