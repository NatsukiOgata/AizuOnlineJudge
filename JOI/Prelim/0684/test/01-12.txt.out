2
5
6
9
11
12
13
16
17
20
25
26
32
33
34
39
40
41
46
48
50
51
52
56
62
65
66
68
71
73
80
81
82
83
92
96
99
100
